be/src/vec/exec/format/parquet/vparquet_column_chunk_reader.cpp

```diff
@@ -43,11 +44,12 @@ Status ColumnChunkReader::init() {
     // get the block compression codec
     RETURN_IF_ERROR(get_block_compression_codec(_metadata.codec, _block_compress_codec));
 
+    VLOG_DEBUG << "initColumnChunkReader finish";
     return Status::OK();
 }
 
 Status ColumnChunkReader::next_page() {
-    RETURN_IF_ERROR(_page_reader->next_page());
+    RETURN_IF_ERROR(_page_reader->next_page_header());
     _num_values = _page_reader->get_page_header()->data_page_header.num_values;
```

Comment:
Better rename `_num_values` to `_left_num_values`.

Reply:
remaining